### PR TITLE
[StatusItem] - add component

### PR DESCRIPTION
**Pull request by pietro-maximoff**
https://sovryn.atlassian.net/browse/SOV-565

### CODE REVIEW

File: packages/ui/src/2_molecules/StatusItem/StatusItem.module.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.statusItem {
+  @apply flex items-center;
+  & > span {
+    &:first-child {
+      @apply mr-2;
+    }
+  }
+}
```
Comment (Rickk137) [inline packages/ui/src/2_molecules/StatusItem/StatusItem.module.css]: `.statusItem {
  @apply flex items-center;
  
  & > span {
    &:first-child {
      @apply mr-2;
    }
  }
}`

File: packages/ui/src/2_molecules/StatusItem/StatusItem.test.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  );
+  expect(getByText('Label')).toBeDefined();
+});
+
+test('renders success status item', () => {
+  const { findAllByRole } = render(
+    <StatusItem content="1" label="Label" status={StatusEnum.success} />,
+  );
+  expect(findAllByRole('svg[data-icon="success-icon"]')).toBeDefined();
+});
```
Comment (Rickk137) [inline packages/ui/src/2_molecules/StatusItem/StatusItem.test.tsx]: please wrap this with a describe and component name

File: packages/ui/src/2_molecules/StatusItem/StatusItem.types.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+export enum StatusEnum {
```
Comment (Rickk137) [inline packages/ui/src/2_molecules/StatusItem/StatusItem.types.ts]: Don't use enum in an enum name

File: packages/ui/src/2_molecules/StatusItem/StatusItem.utils.ts [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import { StatusEnum } from './StatusItem.types';
+
+export const getStatusIcon = (status: StatusEnum) => {
+  switch (status) {
+    case StatusEnum.success:
+      return 'success-icon';
+    case StatusEnum.error:
+      return 'failed-tx';
+    case StatusEnum.pending:
+      return 'pending';
```
Comment (Rickk137) [inline packages/ui/src/2_molecules/StatusItem/StatusItem.utils.ts]: use IconNames

File: packages/ui/src/2_molecules/StatusItem/StatusItem.utils.ts [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+export const getStatusClass = (status: StatusEnum) => {
+  switch (status) {
+    case StatusEnum.success:
+      return 'text-success';
+    case StatusEnum.error:
+      return 'text-error';
+    case StatusEnum.pending:
+      return 'text-sov-white';
+  }
+};
```
Comment (Rickk137) [inline packages/ui/src/2_molecules/StatusItem/StatusItem.utils.ts]: all classNames should be in the module css

**Review by Rickk137 — DISMISSED** [review]

**Comment by Rickk137** [comment]
Screen Shot 2022-10-27 at 5 54 49 PM
>
@pietro-maximoff Failed color is different from the design

**Comment by pietro-maximoff** [comment]
> Screen Shot 2022-10-27 at 5 54 49 PM
> 
> > 
> 
> @pietro-maximoff Failed color is different from the design

Gilad added fixes to figma

**Review by soulBit — APPROVED** [review]

**Review by creed-victor — APPROVED** [review]

**Pull request merged**